mnelab/dialogs/cropdialog.py

```diff
@@ -14,6 +14,7 @@ def __init__(self, parent, start, stop):
         self.setWindowTitle("Crop data")
 
         form = QFormLayout(self)
+        form.setAlignment(Qt.AlignCenter)
```

Comment:
Can you try this?

Reply:
I think we shouldn't mess around with the layouts too much. Let's only use `QFormLayout` for the simplest dialogs. BTW, KDE also aligns the labels to the right.